cirq/google/sim/xmon_simulator.py

```diff
@@ -53,6 +54,41 @@
 from cirq.study import ParamResolver, Sweep, Sweepable, TrialResult
 
 
+def pretty_state(state, decimals=2):
+    """
+    Returns the wavefunction as a string in Dirac notation.
+
+    For example:
+
+    Q1 = cirq.GridQubit(0, 0)
+
+    state = np.array([1/np.sqrt(2), 1/np.sqrt(2)], dtype=np.complex64)
+    simulator = cirq.google.XmonSimulator()
```

Comment:
it's still strange to reference the simulator here.  I would just shorten to document what happens on the input:

```
state = np.array([1/np.sqrt(2), 1/np.sqrt(2)], dtype=np.complex64)
print(pretty_state(state)) -> 0.71|0⟩ + 0.71|1⟩
```